thirdparty/DiskANN/include/utils.h

```diff
@@ -596,25 +590,24 @@ namespace diskann {
              << " while expected size is  " << expected_actual_file_size
              << " npts = " << npts << " dim = " << dim
              << " size of <T>= " << sizeof(T) << std::endl;
-      diskann::cout << stream.str() << std::endl;
+      LOG(ERROR) << stream.str();
```

Comment:
Why we need stream here?

Reply:
From original diskann, just for concatenating the strings that we need.